server/dataone_upload.py

```diff
@@ -316,6 +327,51 @@ def create_upload_tale_yaml(tale, remote_objects, item_ids, user, client):
     return pid, len(yaml_file)
 
 
+def upload_license_file(client, license_id):
+    """
+    Upload a license file to DataONE.
+
+    :param client: The client that interfaces DataONE
+    :param license_id: The ID of the license (see `ExtraFileNames` in constants)
```

Comment:
For these license IDs, it looked like you went with arbitrary integers which is a bit opaque to another developer. What do you think about instead using the [SPDX identifier] for the license as its ID which all application code can reference?